include/fc/log/logger.hpp

```diff
@@ -133,6 +133,10 @@ namespace fc
       (fc::logger::get(DEFAULT_LOGGER)).log( FC_LOG_MESSAGE( error, FORMAT, __VA_ARGS__ ) ); \
   FC_MULTILINE_MACRO_END
 
+// Deep mind specialized logging statement, assumes received LOGGER is NOT nullptr!
+#define dmlog( LOGGER, FORMAT, ... ) \
+  (LOGGER)->log( FC_LOG_MESSAGE( info, FORMAT, __VA_ARGS__ ) )
+
```

Comment:
I see no reason for any of the changes in this file including the addition of this. Now that this is like any other appender you can just use `fc_ilog`

Reply:
The idea here was to differentiate `dmlog` macro from other standard macros, so that call sites does not look like normal "logging" output. Furthermore, this macro disables the `if` statement since it's expected already that each call site validates that the logger is set to something valid for deep mind to work.

I would prefer to keep it personally, so implementation is clearer (IMO), but I'm not strongly against switching to `fc_ilog` if you think it's unacceptable to have the `dmlog` macro here.

Reply:
My personal preference is to not create a new macro and just use `fc_dlog(*logger, ...)` everywhere. But if you really want to keep the `dmlog` macro then I do think that the macro belongs in the nodeos code and not here in fc.

Reply:
Either way I would like the macro to check logger availability instead at every usage of macro.

Reply:
I'll remove `dmlog` and use an existing macro, `dmlog` is not making unanimity :D 

That will fix both of your concerns.